src/generators/java/mekanism/generators/common/config/GeneratorsConfig.java

```diff
@@ -101,6 +106,16 @@
               .defineList("windGenerationDimBlacklist", new ArrayList<>(), o -> o instanceof String && ResourceLocation.tryCreate(((String) o).toLowerCase(Locale.ROOT)) != null));
         builder.pop();
 
+        builder.comment("Fusion Settings").push(FUSION_CATEGORY);
+        fusionThermocoupleEfficiency = CachedDoubleValue.wrap(this, builder.comment("The fraction of the heat dissipated from the case that is converted to RF.")
```

Comment:
Joules not RF, we don't even support RF anymore the other system of energy we can support based on configs is FE.

Reply:
I'll just change it to Joules and fix the merge conflict.